### PR TITLE
Symmetrizable property and theorems

### PR DESCRIPTION
**Pull request by david20000813**
Adding the property "symmetrizable" and the relevant theorems, as discussed in #923

### CODE REVIEW

File: theorems/T000628.md
Comment (david20000813) [inline theorems/T000628.md:14]:
```suggestion
See Theorem 1.2.8 in {{doi:10.2991/978-94-6239-216-8}}.
```

File: theorems/T000627.md
Comment (danflapjax) [inline theorems/T000627.md:6]:
```suggestion
  - P000198: true
```
"$\aleph_1$-compact" in the source is exactly P198.

File: theorems/T000627.md
Comment (david20000813) [inline theorems/T000627.md:14]:
```suggestion
See Theorem 1.2.5 in {{doi:10.2991/978-94-6239-216-8}}.
```

**Comment by david20000813** [comment]
@danflapjax Thanks for spotting that!

File: properties/P000205.md
Comment (prabau) [inline properties/P000205.md:18]:
```suggestion
being contained in $U$.

Equivalently, there exists a symmetric $d$ such that $A \subseteq X$ is closed iff $d(x,A)=\inf_{a \in A} d(x,a) > 0$ for all $x \in X \setminus A$.
```
for readability

**Comment by prabau** [comment]
To parallel what's in P205, the README for P102 (semimetrizable) should also contain some explanation of the relationship with symmetrizable and the variant terminology in the literature.

I am reading both properties, trying to make sense of things without having previous knowledge of these concepts, and I find the second bullet for P102 to be confusing.  What are these $G_n(x)$?  Are they meant to be open sets, or neighborhoods of $x$ or just any sets containing $x$?  Some clarifications would be useful I think.

**Comment by prabau** [comment]
Question: For sources like Wilson that use "semimetrizable" to mean "symmetrizable", is it the case that they don't consider P102 at all in their paper?

**Comment by prabau** [comment]
P102: to better show the parallel with P205, we can change the first bullet from
`... $p$ lies in the closure of some $Y\subseteq X$ iff $\inf\{d(y,p):y\in Y\}=0$.`
to
`... $p$ lies in the closure of some $Y\subseteq X$ iff $d(p,Y)=\inf\{d(p,y):y\in Y\}=0$.`

**Comment by david20000813** [comment]
> Question: For sources like Wilson that use "semimetrizable" to mean "symmetrizable", is it the case that they don't consider P102 at all in their paper?

That is the case, yes.

**Comment by david20000813** [comment]
> P102: to better show the parallel with P205, we can change the first bullet from `... $p$ lies in the closure of some $Y\subseteq X$ iff $\inf\{d(y,p):y\in Y\}=0$.` to `... $p$ lies in the closure of some $Y\subseteq X$ iff $d(p,Y)=\inf\{d(p,y):y\in Y\}=0$.`

Agree.

**Comment by prabau** [comment]
P102: The second bullet references zb:0116.14303, but it looks like it should be zb:0113.16501.
Theorem 3.2 in that paper also says that the $G_n(x)$ should be open sets.  So that should be added ("... non-increasing sequence of open sets ...").

zb:0116.14303 is mentioned in the first paragraph ("Note that ...").  So it should also be added to the `refs:` section.

File: properties/P000102.md
Comment (prabau) [inline properties/P000102.md:24]:
```suggestion
- $X$ is {P2}. And, for each $x \in X$, there is a non-increasing sequence of open sets $\{G_n(x):n<\omega\}$ forming a local neighborhood basis for $x$. Moreover, if for some sequence $x_n$ in $X$, $y\in X$ belongs to $G_n(x_n)$ for all $n$, then $x_n\to y$. In particular, $X$ is {P28}. This equivalence is shown in Theorem 3.2 of {{zb:0113.16501}}.
```

**Comment by david20000813** [comment]
@prabau Implemented. I got confused between the two references. They were both written by Heath, and zb:0116.14303 also includes the result, but it was actually proved in zb:0113.16501.

File: properties/P000102.md
Comment (prabau) [inline properties/P000102.md:8]:
```suggestion
  name: A regular semi-metric space for which there is no semimetric under which all spheres are open (Heath)
```

**Comment by prabau** [comment]
Sorry one more comment.  Is the definition of a "symmetric" just a function $d$ ... that is reflexive and symmetric, or does it also require the extra condition on the open sets?  Asking because the if it's with the condition on open sets, the equivalent condition in the following paragraph is kind of not right.  Does something need to be rephrased somewhere?

And similarly for "semimetric".

**Comment by david20000813** [comment]
@prabau Symmetric is supposed to be a function that’s just symmetric and reflexive. Then a space is symmetrizable if there’s a symmetric generating the topology in the specified way. (And any symmetric generates a topology in exactly that manner.) It’s similar to metric, where a metric is just a function and a metrizable space is a space whose topology is generated by a metric.

Semimetric is more confusing. DOI: 10.2991/978-94-6239-216-8 defines a semimetric to be a symmetric that is compatible with the topology in the specified manner, so a semimetric, under that definition, is supposed to be not just a function. If we’re strictly following DOI: 10.2991/978-94-6239-216-8, then we should say “symmetric” in both the definition and the equivalent condition, and say a symmetric satisfying the compatibility with topology condition to be a semimetric. But DOI: 10.2307/2370790 just defines a semimetric as a function, the exact same as a symmetric.

**Comment by prabau** [comment]
> @prabau Symmetric is supposed to be a function that’s just symmetric and reflexive. Then a space is symmetrizable if there’s a symmetric generating the topology in the specified way. (And any symmetric generates a topology in exactly that manner.) It’s similar to metric, where a metric is just a function and a metrizable space is a space whose topology is generated by a metric.

Thanks, your explanation makes that very clear.


> Semimetric is more confusing. DOI: 10.2991/978-94-6239-216-8 defines a semimetric to be a symmetric that is compatible with the topology in the specified manner, so a semimetric, under that definition, is supposed to be not just a function. If we’re strictly following DOI: 10.2991/978-94-6239-216-8, then we should say “symmetric” in both the definition and the equivalent condition, and say a symmetric satisfying the compatibility with topology condition to be a semimetric. But DOI: 10.2307/2370790 just defines a semimetric as a function, the exact same as a symmetric.
> 
> I’ll rephrase to make things follow DOI: 10.2991/978-94-6239-216-8, and just mentions other sources call just a function a semimetric.

About semimetric, I don't know if we want to complicate things even more for the reader.  Maybe things as they are are ok, given that the reader will follow some of the sources to clarify anything???

File: properties/P000205.md
Comment (prabau) [inline properties/P000205.md]:
```suggestion
there is a "symmetric" $d$ (i.e., a function $d:X^2\to[0,\infty)$
satisfying reflexivity ($d(x,y)=0\Leftrightarrow x=y$) and symmetry
($d(x,y)=d(y,x)$)), such that a subset $U \subseteq X$ is open iff for each $x \in U$,
```
```suggestion
there is a "symmetric" $d$, i.e., a function $d:X^2\to[0,\infty)$
satisfying reflexivity ($d(x,y)=0\Leftrightarrow x=y$) and symmetry
($d(x,y)=d(y,x)$), such that a subset $U \subseteq X$ is open iff for each $x \in U$,
```
Adding two parentheses will make clear what a symmetric is actually.

Comment (prabau) [reply properties/P000205.md]: Your explanation is even better.  I'll delete this comment.

**Comment by david20000813** [comment]
@prabau Ah, sorry, I saw your comment after I've already finished rewording stuff. But perhaps you might find the current version clearer?

**Comment by prabau** [comment]
Thanks. Very clear now.

All good for me.  I'll let @danflapjax do the final review.

**Comment by danflapjax** [comment]
This is admittedly frivolous, but do you mind if I change the ID of symmetrizable to P104 to be near semimetrizable? I think it's slightly nicer for navigation, and I would find it more aesthetically pleasing.

**Comment by david20000813** [comment]
Out of curiosity, why did this not happen to P47 and P48...?

**Review by danflapjax — APPROVED** [review]

**Comment by danflapjax** [comment]
My hypothesis is that the names semimetrizable and symmetrizable are similar enough that the fuzzy search gives a boost in the search to the property listing both, but that's just a guess. I don't actually know how the tie is broken between P47 and P48.

**Comment by StevenClontz** [comment]
This is related to https://topology.pi-base.org/spaces?q=Semi-Hausdorff matching US. I'll investigate now.

**Comment by StevenClontz** [comment]
As of https://github.com/pi-base/web/pull/197/ we now have https://topology.pi-base.org/spaces?q=Semi-Hausdorff and (on this branch) https://topology.pi-base.org/spaces?q=Symmetrizable returning the intended results.

**Comment by StevenClontz** [comment]
Merging as @danflapjax has already approved modulo my fix of the app.

**Pull request merged**
